src/main/java/Duke.java

```diff
@@ -1,10 +1,264 @@
+        } else { // if task is event
+            Event actualTask = (Event) task;
+            System.out.println(taskType + " " + status + " " + item + " -> at :" + actualTask.getTime() + "\n");
+        }
+
+        System.out.println("You have a total of " + this.index + " task(s) in your list !\n");
+        System.out.println("<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<\n");
+    }
+
+    public void deleteTask(int index) {
+        Task task = this.ls.get(index);
+        String item = task.getItem();
+        String status = task.getStatus();
+        String taskType = task.getSign();
+        int num = index + 1;
+
+        // remove task from array
+        this.ls.remove(index);
+        this.index--;
+
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n");
+        System.out.println("I've removed this task for you : ");
+        if (task instanceof Todo) { // if task is to-do
+            System.out.println(num + ". " + taskType + " " + status + " " + item + "\n");
+        } else if (task instanceof Deadline) { // if task is deadline
+            Deadline actualTask = (Deadline) task;
+            System.out.println(taskType + " " +  status + " " + item + " -> by :" + actualTask.getDeadline() + "\n");
+        } else { // if task is event
+            Event actualTask = (Event) task;
+            System.out.println(taskType + " " + status + " " + item + " -> at :" + actualTask.getTime() + "\n");
+        }
+        System.out.println("You now have " + this.index + " task(s) in your list !\n");
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n");
+    }
+
+    public static void main(String[] args) throws DukeException {
+
+        String logo = "     .\"\".    .\"\",\n" +
+                "     |  |   /  /\n" +
+                "     |  |  /  /\n" +
+                "     |  | /  /\n" +
+                "     |  |/  ;-._\n" +
+                "     }  ` _/  / ;\n" +
+                "     |  /` ) /  /\n" +
+                "     | /  /_/\\_/\\\n" +
+                "     |/  /      |\n" +
+                "     (  ' \\ '-  |\n" +
+                "      \\    `.  /\n" +
+                "       |      |\n" +
+                "       |      |";
+
+        System.out.println("HIIIIII\n" + logo + "\n");
+
+        Scanner sc = new Scanner(System.in);
+
+        // instantiate duke object
+        Duke klaun = new Duke();
+
+        // introduce duke
+        klaun.introDuke();
+
+        // get input
+        String input = "";
+
+        // checks for next line of input
+        while (sc.hasNextLine()) {
+            input = sc.nextLine();
+
+            try {
+                if (input.equals("bye")) {
+                    klaun.sayBye();
+                    break;
+                } else if (input.equals("list")) { // if user calls for list
```

Comment:
could perhaps assign a variable name firstWord = input.split(" ")[0] so that it can be directly referenced in the rest of the if else statements!